home.css

```diff
@@ -0,0 +1,65 @@
+@import url(dexterous-ui.netlify.app/stylesheets/main.css);
+
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons+Outlined");
+
+.navigation{
+    background-color: #004643;
+    justify-content: space-between;
+}
+
+.banner-image{
+    background-image: url(/assets/gourmet\ bg.jpg);
+    height: calc(100vh - 5rem);
+    background-position: center;
+    background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+
+}
+
+.flex-row{
+    align-items: center;
+    gap: 0;
+}
+
+.btn-link-noBorder{
+    color: #fffffe;
```

Comment:
Just a suggestion, you can use variables for colors

Reply:
fixed it in the latest commit